modules/game_module.rb

```diff
@@ -0,0 +1,81 @@
+require_relative '../classes/author'
+
+module GameModule
+  def create_game
+    publish_date = entering_date
+
+    multiplayer = multiplayer?('Multiplayer?')
+    puts 'Enter last played date (YYYY-MM-DD):'
+    last_played_at = entering_date
```

Comment:
Anytime a user wants to input `last played date` it show;
`Enter last played date(YYYY-MM-DD)` and `Please Enter a date(YYYY-MM-DD)`
Kindly show `Enter last played date(YYYY-MM-DD)` only when a user wants to input the last played date